test_enclave/src/pcl_key.bin

```diff
@@ -0,0 +1 @@
+""33DDUUffww����
```

Comment:
We need a `.gitattributes` file to ensure these types of files aren't treated as text (and potentially munged due to CRLF foolishness). I'd recommend using [the example from mobilecoin.git](https://github.com/mobilecoinfoundation/mobilecoin/blob/master/.gitattributes). This is kind of pulling a thread, because it will require [defining the merge driver as well](https://github.com/mobilecoinfoundation/mobilecoin/blob/master/.gitconfig).

Reply:
Git is so elegant and well designed!

Gotcha, I'll add this...